ADR/ADR025-remove-papertrail.md

```diff
@@ -0,0 +1,21 @@
+# ADR025: Decision Record Title (a description of the decision, not the problem)
+
+Date: 2024-02-01
+
+## Status
+
+> Accepted
+
+## Context
+
+Previously we added [Papertrail](ADR013-use-paper-trail-gem-for-auditing-and-record-backups.md) with the intention to support reinstating from previous versions. We are not using it for that at all and we are not certain we're using it for the right models. Note that the previous ADR was added over a year ago.
```

Comment:
On previous superseded ADRs (e.g. https://github.com/alphagov/forms/blob/main/ADR/ADR003-use-govuk-signon.md) we've changed the old ADR's status to `Superseded` and linked to the new one - would you mind adding that to this PR?

Reply:
Good shout!